examples/tree_2d_dgsem/elixir_euler_slip_wall.jl

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
It feels to me that the only point of this elixir is to create a test for the slip wall BCs for the TreeMesh - the setup itself has no merit by itself. Wouldn't it be possible to reuse an existing setup (say, another weak blast wave elixir) and just replace the BCs in the `test_trixi_include` command?

Reply:
Sure, I can try and reuse the existing elixir to trim the clutter.

Reply:
I am not sure how I can do this. The elixir I borrowed for a lot of this test `examples/tree_2d_dgsem/elixir_euler_ec.jl` uses periodic BCs (the default) so if I set a new set of `boundary_conditions` how would I pass this to the semidiscretization from the command line?

Reply:
In `examples/tree_2d_dgsem/elixir_euler_ec.jl`, you could change
```julia
semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver)
```
to
```julia
semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver,
                                    boundary_conditions=boundary_condition_periodic)
```
And then call `trixi_include` with `boundary_conditions=boundary_condition_slip_wall`. I think this should work.

To be clear: I don't want to make life extra hard for you, it's just that we already have 200+ elixirs and I'd like to avoid adding more if there are reasonable alternatives (which I believe this approach here is, if it works :grimacing:, that is).

Reply:
You also have to set `periodicity=true` for the `mesh` and overwrite this default setting for the test.

Reply:
Oh right, I forgot. Thanks for catching that.

Reply:
Okay, this works. I updated the `elixir_euler_ec.jl` as suggested and now use it for the `boundary_condition_slip_wall` test. Also, the `elixir_euler_slip_wall.jl` is removed from the repo.